Makefile

```diff
@@ -247,32 +260,34 @@ push: ## Push Grapl containers to Docker Hub
 
 .PHONY: up
 up: build-services ## Build Grapl services and launch docker-compose up
-	$(WITH_LOCAL_GRAPL_ENV) \
+	$(WITH_LOCAL_GRAPL_ENV)
 	docker-compose -f docker-compose.yml up
 
 .PHONY: up-detach
 up-detach: build-services ## Docker-compose up + detach and return control to tty
 	# Primarily used for bringing up an environment for integration testing.
-	# Usage: `make up-detach PROJECT_NAME=asdf`
-	$(WITH_LOCAL_GRAPL_ENV) \
+	# For use with a project name consider setting COMPOSE_PROJECT_NAME env var
+	# Usage: `make up-detach`
+	$(WITH_LOCAL_GRAPL_ENV)
 	docker-compose \
-		-p $(PROJECT_NAME) \
-		-f docker-compose.yml \
+		--file docker-compose.yml \
 		up --detach --force-recreate
 
 .PHONY: down
 down: ## docker-compose down - both stops and removes the containers
-	$(WITH_LOCAL_GRAPL_ENV) \
-	docker-compose $(EVERY_COMPOSE_FILE) down --remove-orphans
+	$(WITH_LOCAL_GRAPL_ENV)
+	docker-compose down
+	docker-compose --project-name "grapl-integration_tests" down
+	docker-compose --project-name "grapl-e2e_tests" down
```

Comment:
Might be worth adding `--timeout=0` to these calls, since we don't really care about state.

Extracting the particular project invocations to their own separate targets might make things more composable, and provide for better overall handling of environments. Though perhaps not, since it's currently impossible to have more than one of these environments running at the same time.

At the very least, extracting the _names_ of the environments as top-level makefile variables would be good, since they're repeated throughout a few times. After that, switching the "_" to "-" would be a nice touch.

Reply:
Would `--timeout=0` mean this could hang longer or indefinitely?

Note it is possible to have multiple environments _exist_ at the same time, even though not running. Running `down` will remove all the containers, even if they aren't running. We don't destroy the test containers for the tests in case a developer wants to inspect them after a run.

Reply:
Testing locally, `--timeout=0` results in a very quick shutdown of everything.

When I said "we don't really care about state", I should have said "we don't really care about cleanly shutting down", which is true, right?

Reply:
> Testing locally, --timeout=0 results in a very quick shutdown of everything.

Sold.

> At the very least, extracting the names of the environments as top-level makefile variables would be good, since they're repeated throughout a few times. After that, switching the "_" to "-" would be a nice touch.

I'm fine with moving to top-level variables, but since these values are used outside of the Makefile I'd rather not change them in this PR.